uwblueprint/jump-math
# fix: crashing heroku deployment

## Notion ticket link
n/a



## Implementation description
- [fix: Copy .cjs files to build/ directory after yarn build](https://github.com/uwblueprint/jump-math/pull/378/commits/2291ae1ba067a77a05ee1269402a3d9aea83af86)
- [fix: Remove the need for .cjs extensions for library files](https://github.com/uwblueprint/jump-math/pull/378/commits/be1d1a01cc303d6a30717f39054f83a75fedf9bc)
- [fix: Add eslint rule to catch manual imports of .cjs files](https://github.com/uwblueprint/jump-math/pull/378/commits/3ad3708f9156192c47226b75edcb67da8f2109da)
- [fix: Ignore build directory when using nodemon](https://github.com/uwblueprint/jump-math/pull/378/commits/9b54a78fda1eedca8a4bdc95a1d9032d688c2859)
- [fix: Lint ALL project files (not just .ts/.js files)](https://github.com/uwblueprint/jump-math/pull/378/commits/4bf4e5a9b80d09a879d500edfe15d6a26af6c1e6)



## Steps to test
0. Consider `rm -rf build/` for a fresh start.
1. Run `yarn build` locally to generate your `build/` directory.
2. Run `node build/server.js` and verify that server does not crash due to an import error (it might crash due to missing environment variables).



## What should reviewers focus on?
* 


## Checklist
